TestFileGenerator.js

```diff
@@ -42,7 +42,7 @@ function validateBuilderFunctions(content, theme, version) {
             }
         }
 
-        if (htmlContent.match(/\/\/(?<!:\/\/)(?<!"\/\/)/gm) || htmlContent.includes('/*')) {
+        if (htmlContent.match(/\/\/(?<!:\/\/)(?<!"\/\/)(?<!"sha512-\S*)/gm) || htmlContent.includes('/*')) {
```

Comment:
Now that I'm thinking about more, we went a little wild with that `*`  .  To limit the `//` match to the sha512 value, maybe we should specify the possible length of that value, since we know it (88 characters max).  That way if there is ever stuff AFTER the sha value that has // in it, the error will be picked up.

```suggestion
        if (htmlContent.match(/\/\/(?<!:\/\/)(?<!"\/\/)(?<!"sha512-\S{0,88})/gm) || htmlContent.includes('/*')) {
```

Reply:
Ohh that makes sense yes